locale/justoverclock-flarum-ext-welcomebox

```diff
@@ -0,0 +1,8 @@
+  justoverclock-flarum-ext-welcomebox:
```

Comment:
First line is the same as in the original file, so only flarum-ext-welcomebox, filename would be flarum-ext-welcomebox.yml, and we still need the forum parent key.